examples/functions/trip/TR_ParallelService_OeBB_wrong.xml

```diff
@@ -0,0 +1,243 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<OJP xmlns="http://www.vdv.de/ojp" xmlns:siri="http://www.siri.org.uk/siri" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.1-dev" xsi:schemaLocation="http://www.vdv.de/ojp ../../../OJP.xsd">
+	<!--
+We have a parallel train NJ 490 and NJ 40490
+Changement of NJ 490 to NJ590 at the border to Germany. This should be modeled with ParallelService
```

Comment:
Please add a description here as well why this is "wrong"

Reply:
done